codegen/src/test/resources/software/amazon/awssdk/codegen/poet/eventstream/test-visitor-builder.java

```diff
@@ -67,5 +91,16 @@ public void visit(EventOne event) {
         public void visit(EventTwo event) {
             onEventTwo.accept(event);
         }
```

Comment:
Should we have `visitEventTwo` and `visitEventOne` and deprecate `visit`?

Reply:
I considered this, but I think I don't think that would actually work when you do the dispatch; for example,

```
eventStream.visit(visitor);
```

We don't know if `visit` is implemented, or `visitEventOne` is implemented, or both!

Reply:
Gotcha